requirements/dev.in

```diff
@@ -12,6 +13,7 @@ isort                     # to standardize order of imports
 pip-tools                 # Requirements file management
 pycodestyle               # PEP 8 compliance validation
 pydocstyle                # PEP 257 compliance validation
+simplejson                # Simple JSON to replace python's json as it supports python 3.5 which python's one doesn't
```

Comment:
I'm confused; why would we need this only for local development?  And Python 3.5's json module should certainly support Python 3.5...